packages/usa-file-input/src/usa-file-input.twig

```diff
@@ -28,6 +28,7 @@
     {% if multiple -%} multiple{%- endif -%}
     {% if disabled_state == 'disabled' %} disabled{%- endif %}
     {% if disabled_state == "aria-disabled" -%} aria-disabled="true"{%- endif -%}
+    {% if error_message -%}data-errorMessage="{{ error_message }}"{%- endif -%}
```

Comment:
A couple things here...

## data-errorMessage

This is not a new data attribute. This already exists in the code and used to allow users to put custom "Accepted file message" when a user tries to upload a non-acceptable file type.

We provide guidance to the user to use `data-errorMessage` and changing this would be a breaking change for users:

> **Customizing the error message.** Add the data attribute data-errorMessage to usa-file-input to include a custom error message. [^1]

[^1]: [data-errorMessage guidance](https://arc.net/l/quote/qwrmgyzh)

## error_message control

I was incorrectly using a predefined content name `error_message` to set this value but opted to create a new `accepted_file_message` instead.

- `error_message` is used for the "helpful error message" that appears above the file input
- `accepted_file_message` will be used to set the optional `data-errorMessage` that appears inside the file input when an unacceptable file is selected
- I chose `accepted_file_message` because it matches the class of the displayed message: `usa-file-input__accepted-files-message`

I went ahead and named named both controls so they display cleanly:

![image](https://github.com/user-attachments/assets/336ee556-8ad3-4f93-8ccd-6866f4869439)

Resolved in [655adc3](https://github.com/uswds/uswds/pull/6168/commits/655adc349b3f23b5c90a481f280e543f569a10bf)

Reply:
Thanks @mahoneycm, it makes sense to keep the existing data attribute.

We should make the storybook control label more readable, so I like you proposed change. It still feels slightly confusing to have it be labeled `data-errorMessage` and the control labeled `Accepted files message`, but not a blocker if it's consistent with rendered code.

Reply:
Yeah I agree. I had trouble naming the control because they're technically both error messages. I opted for the classname of the DOM element here. Let me know if you have another idea!